### PR TITLE
Document "Getting Started" section

### PR DESCRIPTION
**Pull request by halfbro**
Addresses #12 

This is some initial documentation on how to start a game of OpenRCT2.

In the future we will want to add a link to the 'Scenario Editor' section, since the built-in method of making a sandbox park uses it.

### CODE REVIEW

**Review by DeerTears — APPROVED** [review]
I read through these and didn't find anything wrong with them. I might add a link to the scenario editor page when it's mentioned at the end, except no such page exists yet.

File: playing/getting_started/index.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Getting Started
+===============
+
+In OpenRCT2, there are two main game modes - Scenario mode and Sandbox mode.
```
Comment (Gymnasiast) [inline playing/getting_started/index.rst:4]: This suggests that there is a technical distinction between the two, which is not the case. There is only scenario mode, although you can adjust the settings in a way that makes it more like a sandbox. Please rephrase this section and the one about "playing a sandbox" to emphasise that better.

A sandbox mode is a nice-to-have that is on our list (though not super high), which is probably why it ended up in the TOC, but the documentation should reflect the _current_ state of the game.

Comment (halfbro) [reply playing/getting_started/index.rst:4]: Sounds good, would you want to just delete the section on Sandboxes then? It doesn't seem to add that much on its own.

Or should I mention something about enabling cheats and getting rid of money/objectives?

Comment (Gymnasiast) [reply playing/getting_started/index.rst]: Yeah, mention the following:
- all games are scenarios with an objective
- a sandbox mode is planned, in the meantime one can use cheats\* to emulate it

and remove the separate section about sandboxes.

\* we should turn that into a link once there is a page about using cheats

File: playing/getting_started/index.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Getting Started
+===============
+
+OpenRCT2 features a number of challenges known as scenarios. Scenarios feature pre-created parks designed to challenge your management skills. Certain parts of the game are constrained, like the land you are allowed to build on, the amount of money you have, or even how high above the ground you are allowed to build!
+
+In addition to constraints in the scenario itself, each scenario has an objective that must be fulfilled. The first objectives are mostly to build parks with a certain rating and number of guests by a certain time period. As you unlock more scenarios, different kinds of objectives become available.
+
+If you are interested in designing your dream park with no constraints or objectives, there is a method to emulate a "sandbox mode" by utilizing cheats added into OpenRCT2.
```
Comment (Gymnasiast) [inline playing/getting_started/index.rst:8]:
```suggestion
If you are interested in designing your dream park with no constraints or objectives, there is a method to emulate a "sandbox mode" by using cheats added into OpenRCT2.
```

File: playing/getting_started/index.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Starting a Scenario
+-------------------
+
+To start a new scenario, click on the "Start a new game" button.
```
Comment (Gymnasiast) [inline playing/getting_started/index.rst:13]:
```suggestion
To start a new scenario, click on the “Start a new game” button.
```

**Pull request merged**